docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -0,0 +1,190 @@
+| 2.68.0         | December 2024    |                  | Minor        | STS          | 2.69.0                  |
+| 2.69.0         | January 2025     |                  | Minor        | STS          | 2.70.0                  |
+| 2.70.0         | February 2025    |                  | Minor        | STS          | 2.71.0                  |
+| 2.71.0         | March 2025       |                  | Minor        | STS          | 2.72.0                  |
+| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
+| 2.72.0         | May 2025         | Yes              | Major        | LTS          | 2.73.0                  |
+
+**Examples:**
+
+* To qualify for support, you must update to 2.63.0 when released if you have Azure CLI 2.62.0 installed.
+* If Azure CLI 2.66.0 is the last release of Azure CLI before breaking change in the fourth quarter of the calendar year, it's supported until the release of Azure CLI 2.79.0.
+
+## Breaking changes
+
+Major version releases of Azure CLI introduce breaking changes and occur up to two times per year. Breaking changes can occur at any point for preview extensions. Non-GA extensions aren't required to adhere to breaking change policies.
+
+For detailed information about breaking changes in Azure CLI, see [Release notes](./release-notes-azure-cli.md).
+
+## Supported environments
+
+The Azure CLI runs on multiple operating systems (OS) and architectures. For a complete list, see [Install the Azure CLI](./install-azure-cli.md). To be supported by Microsoft, the following criteria must be met:
+
+* The version of Azure CLI is supported.
+* The version of the OS is currently in mainstream support by the OS publisher.
+* The dependencies required by the current version of Azure CLI are supported on the OS.
+
+The Azure CLI ends support for a platform when one of the following conditions is met:
+
+* The OS reaches its end of life as defined by the platform owner.
+* The version of Python required by Azure CLI reaches its end of life, is no longer supported on the OS, or has an unfixed critical security issue.
+
+Azure CLI announces the end of support of an OS or one of its dependencies within three months of the public announcement of the retirement.
+
+## Python dependency
+
+Azure CLI depends on Python version 3.8 or above. The following table summarizes the expected end of support for each version of Python:
+
+|Python version|End of support date|
+|-|-|
+|3.8|October 2024 |
+|3.9|October 2025 |
+|3.10|October 2026 |
+|3.11|October 2027 |
+
+Azure CLI can only be supported on operating systems where the above versions of Python are supported.
+
+* **Windows:** The currently supported versions of Windows client and server meet Python version requirements.
+* **macOS:** The currently supported version of macOS 10.9 and above meet Python version requirements.
```

Comment:
macOS 12 is the oldest OS that is not EOL.

Reply:
@bebound, `10.9` has been changed to `12` in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455). Please leave additional comments in the new PR.